### DISCUSSION TITLE
节点加密，忘记密码，如何去除密码

### DISCUSSION THREAD

**Discussion by danhuang2017** [freeplane/freeplane#1807] [Help]
节点加密，忘记密码，如何去除密码

**Comment by danhuang2017** [comment]
Node encryption, forgotten password, how to remove password

**Reply by danhuang2017** [reply]
Who can help me

**Comment by macmarrum** [comment]
Freeplane uses PBEWithMD5AndDES to encrypt Nodes.
→ https://stackoverflow.com/questions/54143361/what-is-pbewithmd5anddes

Therefore, if you have forgotten your password, you would need to break the encryption.
→ https://en.wikipedia.org/wiki/Data_Encryption_Standard#Security_and_cryptanalysis
